homeassistant/components/tradfri/__init__.py

```diff
@@ -26,6 +26,7 @@
 KEY_API = 'tradfri_api'
 CONF_ALLOW_TRADFRI_GROUPS = 'allow_tradfri_groups'
 DEFAULT_ALLOW_TRADFRI_GROUPS = False
+DEFAULT_OBSERVE_TIMEOUT = 3600  # Set default timeout to 1 hour in seconds
```

Comment:
The name suggests that is a default for options. Call it `TIMEOUT_OBSERVE`

Reply:
I'm looking into it, also with @lwis suggestion to make it configurable. 

But thanks for the suggestion.